crawler/src/main/java/com/movie/recommender/crawler/controller/CrawlerController.java

```diff
@@ -26,4 +28,16 @@ public ResponseEntity<String> triggerCrawl() {
             return ResponseEntity.internalServerError().body("An error occurred during the crawl process.");
         }
     }
+
+    @RabbitListener(queues = RabbitMQConfig.QUEUE_NAME)
+    public void listenToCrawlMessages(String message) {
```

Comment:
create a `LocationMessage` class to avoid passing raw strings
include fields for `countryCode` and `cityName`
if we will have time, we will add logic to process movies only for the received cities